src/game.js

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import {makeShuffledDeck} from "./utils"
+
+class Game extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      cardDeck: makeShuffledDeck(),
+      currentCards: [],
+      numOfGames: this.props.games,
```

Comment:
In case we don't have props, I think we could declare a default value here as fallback!

Reply:
There is props for this.

Reply:
Yes, but you cannot assume that the props always are given when someone tries to reuse a component. So therefore, a fallback for this vital state would make sense. Overall we shouldn't initialize state with a props value ideally.